resources/connectors_api/README.md

```diff
@@ -0,0 +1,9 @@
+### Setup
+
+You need to specify values for the following variables:
+
+- `ES_HOST_PORT`: Specifies the host and port of your Elasticsearch instance you want to test against (example: `http://localhost:9200`).
```

Comment:
Can/Do we have it set to default value of `localhost:9200` or is it empty by default?

Reply:
Oh I see 
```
"variable": [
		{
			"key": "ES_HOST_PORT",
			"value": "localhost:9200"
		}
	]
}
```
👍